### PR TITLE
Update launchpad relay

### PR DESCRIPTION
**Pull request by jkrup**
Update relay to work with latest pixelJS changes

### CODE REVIEW

File: src/main/java/com/snapchat/launchpad/conversion/ConversionController.java [hunk 1]
```diff
@@ -52,6 +52,66 @@ public ResponseEntity<String> handlePostCapiRequest(
...
+        } catch (Exception e) {
+            logger.error("Failed to handle request...", e);
+            return ResponseEntity.internalServerError().body(e.getMessage());
+        }
+        return ResponseEntity.ok().body(res);
+    }
+
+    private ConcurrentHashMap<String, CachedResponse> cache = new ConcurrentHashMap<>();
+
+    @GetMapping("/config/{tld}/{uuid}.{ext}")
```
Comment (hlee6-sc) [inline src/main/java/com/snapchat/launchpad/conversion/ConversionController.java:83]: does this not need to follow the format
```
@GetMapping(
            value = {""/config/{tld}/{uuid}.{ext}""},
            method = RequestMethod.GET)
    @ResponseBody
```

Comment (jkrup) [reply src/main/java/com/snapchat/launchpad/conversion/ConversionController.java:83]: `@GetMapping` is basically shorthand for `@RequestMapping(method=RequestMethod.GET` is my understanding, but I could stick to the latter for consistency if you prefer

File: src/main/java/com/snapchat/launchpad/conversion/services/RelayService.java [hunk 2]
```diff
@@ -107,10 +107,25 @@ private JsonNode addAdditionalRelayInfo(
         final JsonNode headersNode = objectMapper.valueToTree(headers.toSingleValueMap());
         ((ObjectNode) body).set("headers", headersNode);
 
+        ObjectNode launchpadNode = objectMapper.createObjectNode();
```
Comment (wk93210) [inline src/main/java/com/snapchat/launchpad/conversion/services/RelayService.java:110]: Is it worth defining a schema class for launchpadNode?

File: src/main/java/com/snapchat/launchpad/conversion/ConversionController.java (hunk 1 above)
Comment (hlee6-sc) [reply src/main/java/com/snapchat/launchpad/conversion/ConversionController.java:83]: oh i see i didn't know 👍  nah i think thats fine

**Review by wk93210 — APPROVED** [review]

**Pull request merged**